.github/scripts/runner_determinator.py

```diff
@@ -436,6 +444,14 @@ def get_runner_prefix(
             )
             continue
 
+        if opt_out_experiments:
+            if experiment_name in opt_out_experiments:
+                opt_out_exp_list = ", ".join(opt_out_experiments)
+                log.info(
+                    f"Skipping experiment '{experiment_name}', as this workflow has opted-out: {opt_out_exp_list}"
```

Comment:
Nit, to clarify logs

```suggestion
                    f"Skipping experiment '{experiment_name}', as this workflow has opted-out (opted out experiments are: {opt_out_exp_list})"
```